tests/integration/helpers/tls.py

```diff
@@ -145,6 +180,42 @@ async def check_tls(
         return False
 
 
+async def cannot_connect_without_tls(
```

Comment:
This is a matter of taste ultimately, personally I find it easier to check against and reason with positive functions than with negatives